core/src/utils/account_id.rs

```diff
@@ -105,19 +106,29 @@ impl AccountId32 {
 }
 
 /// An error obtained from trying to interpret an SS58 encoded string into an AccountId32
-#[derive(Clone, Copy, Eq, PartialEq, Debug, Display)]
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
 #[allow(missing_docs)]
 pub enum FromSs58Error {
-    #[display(fmt = "Base 58 requirement is violated")]
     BadBase58,
-    #[display(fmt = "Length is bad")]
     BadLength,
-    #[display(fmt = "Invalid checksum")]
     InvalidChecksum,
-    #[display(fmt = "Invalid SS58 prefix byte.")]
     InvalidPrefix,
 }
 
+impl Display for FromSs58Error {
+    fn fmt(
+        &self,
+        f: &mut scale_info::prelude::fmt::Formatter<'_>,
```

Comment:
I don't understand why you use fmt from scale info?

```suggestion
        f: &mut core::fmt::Formatter<'_>,
```

Reply:
done